upgrades/upgrade_testing_framework/workflows/__init__.py

```diff
@@ -0,0 +1,12 @@
+from upgrade_testing_framework.steps import *
```

Comment:
Chris said that "*" imports are evil and he should change this.

Reply:
Thanks Omar!  I'll fix it in the next commit.